src/routes/Settings.tsx

```diff
@@ -93,7 +93,7 @@ export const SettingsRoute: React.FC = () => {
         />
         <FieldCheckbox
           name="onClickMarkAsRead"
-          label="On Click, Mark as Read"
+          label="Mark as read on click"
```

Comment:
Should we keep this capitalized? i agree with making it a single no-comma sentence though

Reply:
I agree with sentence case to match the other fields. 😊 

<img width="244" alt="Screen Shot 2022-09-23 at 17 57 00" src="https://user-images.githubusercontent.com/5850625/192013146-28f124d4-4714-4529-b58c-35a8c51a39db.png">